node/src/main/scala/coop/rchain/node/configuration/commandline/Options.scala

```diff
@@ -175,6 +175,26 @@ final case class Options(arguments: Seq[String]) extends ScallopConf(arguments)
       "Deploy a Rholang source file to Casper on an existing running node. " +
         "The deploy will be packaged and sent as a block to the network depending " +
         "on the configuration of the Casper instance.")
+
+    val addressCheck: String => Boolean = addr =>
+      addr.startsWith("0x") && addr.drop(2).matches("[0-9a-fA-F]+")
+    val from = opt[String](
+      descr = "Purse address that will be used to pay for the deployment.",
+      validate = addressCheck
+    )
+
+    val phloLimit = opt[Int](
+      descr = "The amount of phlo to use for the transaction (unused phlo is refunded)."
+    )
+
+    val phloPrice = opt[Int](
+      descr = "The price of gas for this transaction in phlo."
+    )
+
+    val nonce = opt[Int](
+      descr = "This allows to overwrite your own pending transactions that use the same nonce."
```

Comment:
We'll have to think about this a little. I know that it comes from the link I sent you for Ethereum, but perhaps we will not implement this piece of logic in rchain.